doc/previewflag.md

```diff
@@ -0,0 +1,18 @@
+# ASAB WebUI PreviewFlag component
+
+`PreviewFlag` is a react reusable component.
+It is used to display a customizable (preview) flag. 
+
+# Setup
+
+To use the preview flag, pass the `name` prop to the component.
+
+# Usage
+
+```
+import { PreviewFlag } from 'asab-webui';
```

Comment:
@queuing4oranges I believe that the usage and import is a bit different. What we want to teach devs is to use the method/component for adding a custom flag correctly, so here it will be something like:

```
function SomeContainer(props){
	...

	props.app.setFlag(t("Preview"));
	
	...
}
```

This is I believe a correct usage. It will call the method `setFlag` from Application object and set the custom flag name for the particular container (this is intended behavior). If you will try it the way you have described, it will likely fail.